replies.go

```diff
@@ -165,14 +165,14 @@ var replies = []Reply{
 	},
 	{
 		pattern: `(impact.+(1\.8|1\.7))|((1\.8|1\.7).impact)`,
-		message: "Impact for older versions is no longer availible to comply with Mojang's EULA.",
+		message: "Impact for older versions of Minecraft (pre 1.8) is no longer availible to comply with Mojang's EULA.",
 	},
 	{
 		pattern: `(modpack|\bftb\b|rlcraft|skyfactory|valhelsia|pixelmon|sevtech)`,
 		message: "Impact is generally incompatible with modpacks and support will not be provided if you encounter bugs with them. It's likely your game will just crash on startup.",
 	},
 	{
-		pattern: `good bot`,
-		message: "thank you *nuzzles*",
+		pattern: `((good)\s*(bots?|human))`,
```

Comment:
This is my main concern, there is probably better RegEx for this.

Reply:
This is a tiny bit shorter:
```suggestion
		pattern: `(good\s*(bot|human))`,
```